actions/java-gradle-publish/action.yaml

```diff
@@ -0,0 +1,100 @@
+name: "Publish java artifacts"
```

Comment:
I'd split this into two actions, one for the nexus release and one for Gradle's plugin portal

Reply:
+1